week-1-ts-ecosystem/w01e01-typescript-basics/src/models/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './user';
+export * from './address';
```

Comment:
Osobiście nie jestem zwolennikiem `barrel'owania` importów w index.ts 'ach

Jest to konieczne w niektórych przypadkach produkowania typów w osobnych `lib`'ach.
Natomiast tutaj przynosi więcej `complexity` niż pożytku.

Reply:
przyzwyczajenie z roboty, wszyscy klepią choć już nie pamiętają po co heh